src/components/ModiteImage/index.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { FunctionComponent, ReactNode } from 'react'
+// @ts-ignore
```

Comment:
just to confirm, does this mean we are ignoring linter in the entire file ?

Reply:
single line